src/scripts/discord.js

```diff
@@ -32,6 +32,7 @@ const observer = (event, arg) => {
           startTimestamp: parseInt(now),
           endTimestamp: parseInt(remaining),
           largeImageKey: mediaInfoModule.mediaInfo.image,
+          largeImageText: (mediaInfoModule.mediaInfo.album) ? mediaInfoModule.mediaInfo.album : 'TIDAL HiFi',
```

Comment:
According to this line it will default to TIDAL HiFi but I think we should keep the original line which is currently used by the app and also used by the idle status

```js
Tidal HiFi ${app.getVersion()}
```